packages/cli/src/cmds/dev/options.ts

```diff
@@ -51,6 +53,20 @@ const devOwnOptions: ICliCommandOptions<IDevOwnArgs> = {
     default: "http://127.0.0.1:9596",
     type: "string",
   },
+
+  mode: {
```

Comment:
This arguments names are too generic. A beacon node uses many "mode"s and "url"s, the name must include a reference to the remote signer

Reply:
Renamed them to signingMode and signingUrl
https://github.com/ChainSafe/lodestar/pull/3502/commits/f9187a64ac045ed2844fac8ec1d68b85731a07ed